pageserver/src/tenant/timeline.rs

```diff
@@ -1187,7 +1187,7 @@ impl Timeline {
         }
     }
 
-    pub(crate) const MAX_GET_VECTORED_KEYS: u64 = 32;
+    pub(crate) const MAX_GET_VECTORED_KEYS: u64 = 128;
```

Comment:
Can you please make this configurable via pageserver config and set the default to `32`?
Issue is that `MAX_GET_VECTORED_KEYS` is used in a bunch of places and I don't want this PR to come with a default behaviour change.

I know you suggested this in the issue, but I didn't consider those users.

Reply:
On it! 

I thought that a read will only _try_ to fill as many keys as up to `MAX_GET_VECTORED_KEYS` but it shouldn't wait until _all_ the keys are filled.